RadixWallet/Clients/AccountsClient/AccountsClient+Live.swift

```diff
@@ -54,19 +54,19 @@ extension AccountsClient: DependencyKey {
 			},
 			saveVirtualAccounts: saveVirtualAccounts,
 			getAccountByAddress: { address in
-				try await getProfileStore().network().entity(address: address)
+				try await profileStore.network().entity(address: address)
 			},
 			hasAccountOnNetwork: { networkID in
 				do {
-					let network = try await getProfileStore().profile.network(id: networkID)
+					let network = try await profileStore.profile.network(id: networkID)
 					// N.B. `accounts` is NonEmpty so `isEmpty` should always evaluate to `false`.
 					return !network.accounts.isEmpty
 				} catch {
 					return false
 				}
 			},
 			updateAccount: { updatedAccount in
```

Comment:
This one might also be a bit risky, if someone calls it with a large number of accounts it will probably break TCA sending. Perhaps we could have `updateAccounts` instead? Or at least add a yield.

Reply:
Yes let's use a bulk add.

Reply:
it is not used anywhere currently with a collection of changes. So it is more a tip for @GhenadieVP in his Hide Accounts PR to use a bulk and yes he is already doing that:

https://github.com/radixdlt/babylon-wallet-ios/pull/871/files#diff-e0bc333f55b4429f352323b6796ac0bab3ed78975edea4e7ca17523620f26da9R22

it is a single call for `N` many accounts, not `N` many calls.

Reply:
Yeah, I just want to discourage people getting into that situation in the future.